operator/main.go

```diff
@@ -150,6 +150,8 @@ func init() {
 	flags.BoolVar(&enableCNPNodeStatusGC, "cnp-node-status-gc", true, "Enable CiliumNetworkPolicy Status garbage collection for nodes which have been removed from the cluster")
 	flags.DurationVar(&ciliumCNPNodeStatusGCInterval, "cnp-node-status-gc-interval", time.Minute*2, "GC interval for nodes which have been removed from the cluster in CiliumNetworkPolicy Status")
 
+	flags.DurationVar(&cnpStatusUpdateInterval, "cnp-status-update-interval", 100*time.Millisecond, "interval between CNP status updates sent to the k8s-apiserver per-CNP")
```

Comment:
Is this time correct? 100 milliseconds?

Reply:
Yes. This is used to ratelimit the number of updates to K8s apiserver per CNP to at most 100 milliseconds. I figured it was a sane default. Is there a specific problem you see with this value?

Reply:
This means we will push every 100 milliseconds, I think it's too aggressive. We would need to get more data to get a sane value, but given the status update can will be read by humans I think they won't be checking every 100 miliseconds to see if the status has changed. Perhaps 1 seconds should be sufficient.